source/libtomo/gridrec/CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+# If on Windows, build gridrec.cc
+if(WIN32)
+# Locate sources for this project headers are included for IDEs
+file(GLOB gridrec_tomo_sources
+    ${CMAKE_CURRENT_LIST_DIR}/cxx/*.cc)   
+
+else(WIN32)
+
+file(GLOB gridrec_tomo_sources
+    ${CMAKE_CURRENT_LIST_DIR}/c/*.c)  
+
+endif(WIN32)
```

Comment:
This statement should be guarded with the `TOMOPY_CXX_GRIDREC` variable from `/cmake/Options.cmake` instead.